amp/WEB-INF/src/org/digijava/module/aim/util/AuditLoggerUtil.java

```diff
@@ -470,8 +483,10 @@ public static void deleteLogsByPeriod(String interval) {
         query.append(" left join amp_audit_logger aal on ");
         query.append(" (aal.objecttype ='org.digijava.module.aim.dbentity.AmpActivityVersion' ");
         query.append("  and cast (aal.objectId AS INTEGER) = aav.amp_activity_id");
-        if (userid != null) {
-            query.append("  and aal.userid=:userId ");
+        query.append("  and aal.modifyDate > :yearToDate");
```

Comment:
what does this line do & why is it added?

Reply:
the requirement says we need to get results for the past 365 days (or one year) this is to filter out events older than 1 year in case the use has not enabled the job to clean the audit logger